test/draw/gpu/GpuSurfaceTest.ooc

```diff
@@ -86,9 +86,6 @@ GpuSurfaceTest: class extends Fixture {
 			gpuImage canvas clear()
 			viewport := gpuImage canvas viewport
 			redBox := IntBox2D new(viewport left, viewport top, viewport width / 2, viewport height / 2)
-			yellowBox := IntBox2D new(viewport width / 2, viewport top, viewport width / 2, viewport height / 2)
-			blueBox := IntBox2D new(viewport left, viewport height / 2, viewport width / 2, viewport height / 2)
-			greenBox := IntBox2D new(viewport width / 2, viewport height / 2, viewport width / 2, viewport height / 2)
```

Comment:
@emilwestergren Maybe you know if these should be included in some test or if it's safe to remove them. They're currently not used.